gpt4all-chat/server.cpp

```diff
@@ -205,6 +205,10 @@ QHttpServerResponse Server::handleCompletionRequest(const QHttpServerRequest &re
     if (body.contains("top_p"))
         top_p = body["top_p"].toDouble();
 
+    float min_p = 1.f;
```

Comment:
This value will keep no tokens by default (minimum likelihood of 100%) - you probably meant `0.f`.

Reply:
Fixed